cmd/bridge/main.go

```diff
@@ -121,22 +122,17 @@ func main() {
 
 	fLoadTestFactor := fs.Int("load-test-factor", 0, "DEV ONLY. The factor used to multiply k8s API list responses for load testing purposes.")
 
-	if err := fs.Parse(os.Args[1:]); err != nil {
+	fs.String("developer-catalog-categories", "", "Allow catalog categories customization. (JSON as string)")
```

Comment:
Why not
```suggestion
	fDevCatalogCategories := fs.String("developer-catalog-categories", "", "Allow catalog categories customization. (JSON as string)")
```

and the below just:
```go
DevCatalogCategories:  fs.fDevCatalogCategories,
```

Reply:
Good catch, changed and tested locally with, and without config file.